configure.ac

```diff
@@ -811,8 +811,15 @@ AC_CHECK_LIB([stdc++], [atoi])
 AC_CHECK_LIB([dl], [dlsym])
 AC_CHECK_LIB([rt], [clock_gettime])
 AC_CHECK_LIB([pthread], [pthread_create])
-#AC_CHECK_LIB([rdmacm], [rdma_create_id]) - doing it here requires rdmacm & libibverbs for udp_lat even on Ethernet only machines
-#AC_CHECK_LIB([ibverbs], [ibv_cmd_poll_cq])
+
+TMP_LDFLAGS=$LDFLAGS
+AC_CHECK_LIB([rdmacm], [rdma_create_id],
+		[LDFLAGS="$LDFLAGS -lrdmacm" VERBS_FLAGS="$VERBS_FLAGS -lrdmacm"],
+		[AC_MSG_FAILURE([rdmacm not found])])
```

Comment:
@rafiw, remove the _action-if-not-found_ section both here and on line 821. It looks to me like this should fix the Jenkins failure with ICC.